site/en/docs/handbook/web-platform/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Web platform docs
+layout: 'layouts/doc-post.njk'
+date: 2022-05-06
+---
+
+## Metadata {: #metadata }
+
+Web Platform documentation should follow the conventions outlined in the [Add a Doc][add-a-doc] guide. 
+
+The `toc.yml` file is located at `site/_collections/_data/docs/web-platform/toc.yml`. 
+Link your new doc in this file in the appropriate category. If your doc moves between Origin and Developer Trials, 
+and when it ships, make a PR to update this file to move it in the list.
+
+The following is a suggested outline when documenting a new Origin or Developer Trial. 
+As the intention is for new web platform features to ultimately be documented over on [MDN][mdn], 
+use the same conventions as described in the [MDN guide to writing API docs][mdn-guide] where possible. 
+
+{% Aside %}
+Once you have written your docs, or if there has been a big change to the feature, write a [quick blog post][blog] to let developers know. 
+The blog post shouldnt repeat these docs, but could share the latest news, or demonstrate part of the feature with a tutorial, 
+then link to the docs for all the details.
+{% endAside %}
+
+## Outline
+
+```md
+
+Intro paragraph explaining briefly what this API is for.
+
+## Current status
+
+Add information about status, and how developers can try out the API. 
+
+## Interfaces
+
+Describe each interface that is part of the API 
+using a description list format.
+
+`interfaceName`
+: Short description.
+
+## Use cases 
+
+Change the heading above to make sense for your content, 
+however here you can add a short explanation, 
+and the core use cases for the API.
+
+## Examples
```

Comment:
```suggestion
## Use cases 

Change the heading above to make sense for your content, 
however here you can add a short explanation, 
and the core use cases for the API.

## Current status

Add information about status, and how developers can try out the API. 

## Interfaces

Describe each interface that is part of the API 
using a description list format.

`interfaceName`
: Short description.

## Examples
```

Reply:
Should we move **Use Cases** right after the **Outline**, so developers can early on decide if this is something they need?

Reply:
I'd like to have the current status above it, because it's not helpful for developers to find something they need then realise they can't use it. So maybe reorder to status, then use cases, then interfaces?

Reply:
Likewise sounds good to me. Deleting the suggestion then.